soc/arm/nordic_nrf/Kconfig.peripherals

```diff
@@ -264,3 +264,12 @@ config HAS_HW_NRF_WDT0
 
 config HAS_HW_NRF_WDT1
 	bool
+
+
+if HAS_HW_NRF_ACL
```

Comment:
When there's only one option inside an `if`, it is better (shorter) to use `depends on` instead of that `if`.

Reply:
Applies to all files

Reply:
Done